stopcovid/dialog/models/state.py

```diff
@@ -45,15 +45,15 @@ def set_language(cls, value: Optional[str]) -> Optional[str]:
 class PromptState(pydantic.BaseModel):
     slug: str
     start_time: datetime.datetime
-    failures: Optional[int] = 0
-    reminder_triggered: Optional[bool] = False
+    failures: int = 0
+    reminder_triggered: bool = False
     last_response_time: Optional[datetime.datetime] = None
 
 
 class DialogState(pydantic.BaseModel):
     phone_number: str
     seq: str
-    user_profile: Optional[UserProfile] = pydantic.Field(
+    user_profile: UserProfile = pydantic.Field(
```

Comment:
This is the biggest change in this whole PR. We assume the user profile is set everywhere and mypy is yelling about it. Looping through the `dialog-state-prod` table there appear to be 0 cases where `user_profile` is set to `None`